changelog.d/20230926_172753_codewithemad_config_loaded_action.md

```diff
@@ -0,0 +1 @@
+- 💥[Feature] New action introduced: CONFIG_LOADED. This actions is called whenever the config is loaded. (by @CodeWithEmad)
```

Comment:
Got a notification for this because I was watching #557, so I took a look. Thanks for adding this! 

One minor note though: are we sure that this should be marked as a breaking change (💥)? It doesn't look to me like this change has the potential to break anything, since at worst it generates a warning — or am I misreading something?

Reply:
You're absolutely right. I'll remove the 💥 flag when I create the new version.

Reply:
Oops. My bad.